private/buf/cmd/buf/command/generate/generatev1.go

```diff
@@ -0,0 +1,174 @@
+// Copyright 2020-2023 Buf Technologies, Inc.
```

Comment:
Why is this file called `generatev1.go`?

Reply:
This file is now removed, as per comment on having one entry point